classes/person.rb

```diff
@@ -8,13 +8,18 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
   def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals < rental
+  end
```

Comment:
- [ ] The add_rental method shpuld be implemented as follows, so as to achieve the has-many association.
![image](https://user-images.githubusercontent.com/81932478/192837502-e1cbf12b-9b58-421d-8e03-d05ded5177aa.png)